.github/workflows/publish-packages.yml

```diff
@@ -48,38 +48,8 @@ jobs:
       - name: Build packages 📦
         run: pnpm packages
 
-      - name: Set @h5web/lib version 📌
-        uses: reedyuk/npm-version@1.1.1
-        with:
-          version: ${{ steps.packageVersion.outputs.PACKAGE_VERSION }}
-          package: 'packages/lib'
-
-      - name: Set @h5web/app version 📌
-        uses: reedyuk/npm-version@1.1.1
-        with:
-          version: ${{ steps.packageVersion.outputs.PACKAGE_VERSION }}
-          package: 'packages/app'
-
-      - name: Set @h5web/h5wasm version 📌
-        uses: reedyuk/npm-version@1.1.1
-        with:
-          version: ${{ steps.packageVersion.outputs.PACKAGE_VERSION }}
-          package: 'packages/h5wasm'
-
-      - name: Publish @h5web/lib 🥳
-        run: cd packages/lib && pnpm publish --access public --provenance --no-git-checks --tag $NPM_TAG
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-          NPM_TAG: "${{ contains(steps.packageVersion.outputs.PACKAGE_VERSION, 'beta') && 'next' || 'latest' }}"
-
-      - name: Publish @h5web/app 🥳
-        run: cd packages/app && pnpm publish --access public --provenance --no-git-checks --tag $NPM_TAG
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-          NPM_TAG: "${{ contains(steps.packageVersion.outputs.PACKAGE_VERSION, 'beta') && 'next' || 'latest' }}"
-
-      - name: Publish @h5web/h5wasm 🥳
-        run: cd packages/h5wasm && pnpm publish --access public --provenance --no-git-checks --tag $NPM_TAG
+      - name: Publish packages 🥳
+        run: pnpm -r publish --access public --provenance --no-git-checks --tag $NPM_TAG
```

Comment:
Just discovered that `pnpm publish` supports the `--recursive` flag. I've checked: it runs only on public packages (without `private: true` in their `package.json`), so it doesn't try to publish the shared package, the Storybook and the demo. :+1: